### ISSUE TITLE
Mangohud does not run in steam flatpak

### ISSUE THREAD

**Issue by Fatmice** [flathub/com.valvesoftware.Steam.Utility.MangoHud#7]
I'm trying to use the mangohud with steam flatpak.  Here are the list of things installed
```
Name                                               Application ID                                     Version            Branch           Origin                 Installation
Discord                                            com.discordapp.Discord                             0.0.12             stable           Flathub                user
GKraken                                            com.leinardi.gkraken                               0.14.4             stable           flathub                user
Steam                                              com.valvesoftware.Steam                            1.0.0.66           beta             flathub-beta           user
MangoHud                                           com.valvesoftware.Steam.Utility.MangoHud                              stable           flathub                user
Freedesktop Platform                               org.freedesktop.Platform                           19.08.12           19.08            Flathub                user
Freedesktop Platform                               org.freedesktop.Platform                           20.08.0            20.08            flathub                user
i386                                               org.freedesktop.Platform.Compat.i386                                  20.08            flathub                user
default                                            org.freedesktop.Platform.GL.default                                   19.08            Flathub                user
default                                            org.freedesktop.Platform.GL.default                                   20.08            flathub                user
mesa-git                                           org.freedesktop.Platform.GL.mesa-git                                  20.08            flathub-beta           user
default                                            org.freedesktop.Platform.GL32.default                                 20.08            flathub                user
mesa-git                                           org.freedesktop.Platform.GL32.mesa-git                                20.08            flathub-beta           user
html5-codecs                                       org.freedesktop.Platform.html5-codecs                                 18.08            Flathub                user
openh264                                           org.freedesktop.Platform.openh264                                     19.08            Flathub                user
openh264                                           org.freedesktop.Platform.openh264                  2.1.0              2.0              flathub                user
GNOME Application Platform version 3.34            org.gnome.Platform                                                    3.34             Flathub                user
GNOME Application Platform version 3.36            org.gnome.Platform                                                    3.36             Flathub                user
```

When I enter the steam flatpak with ```flatpak enter com.valvesoftware.Steam bash -C``` and type ```mangohud``` I get ```bash: mangohud: command not found```

```flatpak override --user --show com.valvesoftware.Steam```
```
[Environment]
MANGOHUD=1
```

What do I need to do to get this working?

**Comment by gasinvein** [comment]
You have Steam `beta` branch installed, thus, app and extension branches don't match. Try with `stable` branch.

**Comment by Fatmice** [comment]
Can it be made to work with either?  I need the ```beta``` branch for aco.

I see the metadata defined as

```
[ExtensionOf]
ref=app/com.valvesoftware.Steam/x86_64/stable
runtime=org.freedesktop.Platform/x86_64/20.08
```

**Comment by gasinvein** [comment]
It could work if MangoHud extension itself is built for `beta` branch. I'll submit it later.

**Comment by Fatmice** [comment]
Okay. I'll try again as soon as it's ready.

**Comment by gasinvein** [comment]
Do you actually need runtime and app `beta`s to use `.GL.mesa-git`, btw? Ping @nanonyme

**Comment by gasinvein** [comment]
Fixed by be2f1e055c396eda40249dc27493cffa5af183da, install `com.valvesoftware.Steam.Utility.MangoHud//beta` once it's published on Flathub.
